build/parser.mjs

```diff
@@ -1080,4 +1083,6 @@ class Parser {
   }
 }
 
-module.exports = new Parser();
+export const parser = new Parser();
+
+export default parser;
```

Comment:
i don't really like doing both

Reply:
me too but i default is priority in the eslint config and i don't really like that, i don't like how the default import works, it opens up the opportunity to have two different words meaning the same thing in the code base or one word meaning two things, that's in module scope.

Reply:
the word default or the name parser? cause if you just export the default, then you can't do a destructure

Reply:
[the export default ](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/prefer-default-export.md)
in the parser it would be okay to remove the default but it wouldn't be in other modules (files)